docs/docs/91-migrations.md

```diff
@@ -12,6 +12,15 @@ Some versions need some changes to the server configuration or the pipeline conf
 - Deprecated alternative names for secrets, use `environment` with `from_secret`
 - Deprecated slice definition for env vars
 - Deprecated `environment` filter, use `when.evaluate`
+- Deprecated some forge specific configs in favor of `WOODPECKER_FORGE_...`:
+  - Use `WOODPECKER_FORGE_URL` instead of	`WOODPECKER_GITHUB_URL`, `WOODPECKER_GITLAB_URL`, `WOODPECKER_GITEA_URL`, `WOODPECKER_BITBUCKET_URL`
+  - Use `WOODPECKER_FORGE_CLIENT` instead of `WOODPECKER_GITHUB_CLIENT`, `WOODPECKER_GITLAB_CLIENT`, `WOODPECKER_GITEA_CLIENT`, `WOODPECKER_BITBUCKET_CLIENT`, `WOODPECKER_BITBUCKET_DC_CLIENT_ID`
+  - Use `WOODPECKER_FORGE_SECRET` instead of `WOODPECKER_GITHUB_SECRET`, `WOODPECKER_GITLAB_SECRET`, `WOODPECKER_GITEA_SECRET`, `WOODPECKER_BITBUCKET_SECRET`, `WOODPECKER_BITBUCKET_DC_CLIENT_SECRET`
+  - Use `WOODPECKER_FORGE_SKIP_VERIFY` instead of `WOODPECKER_GITHUB_SKIP_VERIFY`, `WOODPECKER_GITLAB_SKIP_VERIFY`, `WOODPECKER_GITEA_SKIP_VERIFY`, `WOODPECKER_BITBUCKET_SKIP_VERIFY`
+- Use `WOODPECKER_DEV_OAUTH_HOST` instead of `WOODPECKER_DEV_GITEA_OAUTH_URL`
+- Replace `WOODPECKER_DEV_OAUTH_HOST` with `WOODPECKER_HOST` (public facing address) and `WOODPECKER_WEBHOOK_HOST` (internal webhook address)
+- Deprecated `WOODPECKER_WEBHOOK_HOST` in favor of `WOODPECKER_DEV_WEBHOOK_HOST`
```

Comment:
Why? I use it not in dev/test env.

Reply:
It basically means to newbies of woodpecker don't touch when doing your first setup. It often seams that new users quickly get lost in a jungle of endless guessing when touching more advanced settings.

Reply:
Name them `WOODPECKER_INTERNAL_`? I agree in general to prefix them somehow.

Reply:
Sth like `WOODPECKER_INTERNAL_` or `WOODPECKER_ADVANCED_` would also be fine for me as well. We should just decide on sth unique we can then use for all of those kind of configs.

Reply:
I'm for `internal` then. It is more aligned with deployment architecture, I think about it like "internal network".

Reply:
renamed to internal